src/Sai2Model.cpp

```diff
@@ -1033,6 +1033,44 @@ void Sai2Model::displayLinks() {
 	cout << endl;
 }
 
+void Sai2Model::addLoad(const std::string& link_name,
+						const double& mass,
+						const Vector3d& com_pos,
+						const Matrix3d& inertia,
+						const std::string& body_name) {
+	if (_load_names_to_load_mass_map.find(body_name) != _load_names_to_load_mass_map.end()) {
```

Comment:
Also add a check to make sure that the link name exists, and other ones to make sure the mass and inertia are physically valid (positive mass and positive definite inertia)